tangelo/linq/target/target_cirq.py

```diff
@@ -110,13 +110,36 @@ def simulate_circuit(self, source_circuit: Circuit, return_statevector=False, in
 
         # Run shot by shot and keep track of desired_meas_result only (generally slower)
```

Comment:
I'm not sure this comment matches what's happening down there anymore. I'm actually not sure what is happening down there.